sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/HashExpressionsSuite.scala

```diff
@@ -684,6 +684,21 @@ class HashExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     assert(murmur3HashPlan(wideRow).getInt(0) == murmursHashEval)
   }
 
+  test("SPARK-30633: Use Long seeds for xxHash") {
```

Comment:
This is a good test candidate. Can we extend this test coverage for the other `Murmur3Hash` and `HiveHash`?

Reply:
@dongjoon-hyun, I'm not sure I understand this request. Do we want to have a similar test for Murmur3 and Hive, but where the seeds are 32 bit?